homeassistant/components/deconz/device_trigger.py

```diff
@@ -319,20 +332,26 @@ def _get_deconz_event_from_device_id(hass, device_id):
     device_registry = await hass.helpers.device_registry.async_get_registry()
     device = device_registry.async_get(config[CONF_DEVICE_ID])
 
-    trigger = (config[CONF_TYPE], config[CONF_SUBTYPE])
+    triggers = (config[CONF_TYPE], config[CONF_SUBTYPE])
+
+    triggers = REMOTES[device.model][triggers]
```

Comment:
Still only one trigger
```suggestion
    trigger = REMOTES[device.model][trigger]
```